core/constraints/package_test.go

```diff
@@ -29,5 +29,6 @@ func (*ImportTest) TestImports(c *gc.C) {
 		"core/arch",
 		"core/instance",
 		"core/status",
+		"utils/stringcomparator",
```

Comment:
Adding this import violates the comment above. So we need to adjust the comment to indicate that using the sttringscomparator package is ok.

Reply:
Thanks Ian will update this :)